app/controllers/confirmation_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class ConfirmationController < ApplicationController
+  rescue_from ActiveRecord::RecordNotUnique, with: :error_not_unique
+
+  # This populates instance variables used by the new confirmation form.
+  def new
+    @term = Term.find(params[:term_id])
+    @categories = Category.all
+    @confirmation = Confirmation.new
+  end
+
+  # This receives the submission from the new confirmation form, creating the needed record with the help of various
+  # private methods.
+  def create
+    confirmation = Confirmation.new({
+                                      term_id: params[:term_id],
+                                      user: current_user
+                                    })
+    set_category(confirmation, params[:confirmation][:category])
+    feedback_for(confirmation.save)
+    redirect_to terms_unconfirmed_path
+  end
+
+  private
+
+  # This method takes the result of the confirmation.save directive above and sets an appropriate flash message.
```

Comment:
Minor. Rather than `This method takes the...` I find it easier to read the documentation when it starts with the method name, so my suggestion would be something like `feedback_for takes the...`